src/Gren/Kernel/Scheduler.js

```diff
@@ -49,24 +49,77 @@ function _Scheduler_receive(callback) {
   };
 }
 
+function _Scheduler_concurrent(tasks) {
+  return _Scheduler_binding(function (callback) {
+    if (tasks.length === 0) return callback(_Scheduler_succeed([]));
```

Comment:
Don't think you have to have this inside the binding. Simply return success if `tasks.length === 0`.